includes/polling/toner.inc.php

```diff
@@ -5,12 +5,9 @@
 foreach ($toner_data as $toner) {
     echo 'Checking toner '.$toner['toner_descr'].'... ';
 
-    if ($toner['toner_capacity_oid']) {
-        // FIXME this if can go on 1-Sep-2012
-        $toner['toner_capacity'] = snmp_get($device, $toner['toner_capacity_oid'], '-OUqnv');
-    }
 
-    $tonerperc = round((snmp_get($device, $toner['toner_oid'], '-OUqnv') / $toner['toner_capacity'] * 100));
+    $capacity = $toner['toner_capacity'];
```

Comment:
I'm not sure why you'd set this variable, you're only replacing the new one for toner_capacity so may as well leave it.